_data-prepper/trace-tuning.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Trace tuning
+nav_order: 12
+---
+
+# Trace tuning
+
+Data Prepper for trace analytics in version 0.8.x supports both vertical and horizontal scaling. Adjust the size of your single Data Prepper instance to meet your workload's demands and scale vertically. 
+
+<!--- Is there a better way to phrase the version part of this sentence?--->
+
+Scale horizontally by deploying multiple Data Prepper instances to form a cluster by using the [Core Peer Forwarder](https://github.com/opensearch-project/data-prepper/blob/main/docs/peer_forwarder.md). This enables Data Prepper instances to communicate with instances in the cluster, and is required for horizontally-scaling deployments.
+
+## Scaling tips
+
+The following sections describe useful tips for scaling. We recommend that you modify parameters based on the requirements. We also recommend that you monitor the Data Prepper host metrics and OpenSearch metrics to ensure the configuration is working as expected.
+
+### Buffer
+
+The total number of trace requests that Data Prepper is processing is equal to sum of `buffer_size` in `otel-trace-pipeline` and `raw-trace-pipeline`. The total number of trace requests to OpenSearch is equal to the product of `batch_size` and `workers` in `raw-trace-pipeline`.
+
+<!--- Are each of these items in code blocks buffers? They look like parameters.--->
+
+We recommend that you use the following parameters:
+ * Have the same `buffer_size` value in `otel-trace-pipeline` and `raw-trace-pipeline`
+ * That the values `buffer_size` >= `workers` * `batch_size` in the `raw-trace-pipeline`
+ 
+
+### Workers 
+
+The `workers` setting determines the number of threads that are used by Data Prepper to process requests from the buffer. We recommend that you set the workers based on the CPU utilization. This value can be higher than available processors, as Data Prepper spends significant input/output time when sending data to OpenSearch.
+
+### Heap
+
+Configure the heap of Data Prepper by setting the `JVM_OPTS` environmental variable. We recommend that you set the heap value to a minimum of `4` * `batch_size` * `otel_send_batch_size` * `maximum size of indvidual span`.
+
+As mentioned in the [setup](trace_setup.md#opentelemetry-collector), set `otel_send_batch_size` to a value of `50` in your opentelemetry collector configuration.
+
+<!--- This link doesn't work. Is there a current link we can use? --->
+
+### Disk
+
+Data Prepper uses disk to store metadata required for service-map processing, so we recommend only storing the key fields `traceId`, `spanId`, `parentSpanId`, `spanKind`, `spanName` and `serviceName`. The service-map plugin ensures that it only stores two files with each storing `window_duration` seconds of data. During testing, we found that, for a throughput of `3000 spans/second`, the total disk usage was `4 MB`.
```

Comment:
I feel like we should say "local disk" or "local storage" or "disk space" something like that depending on context. Just saying "disk" feels confusing and vague imo